### ISSUE TITLE
Implementation on react-native-web

### ISSUE THREAD

**Issue by wutayng** [auth0/react-native-auth0#280]
### Description

Unclear implementation for react native web applications.
No tutorials on auth0.com worked for boilerplate react-native web application.
For this application type, loader doesn't seem to be configured for the auth0 module.

### Reproduction

Create react-native-web application from CLI 
- https://github.com/orYoffe/create-react-native-web-app  

Set all URLs in Auth0 to http://localhost:3000
```
npm install react-native-auth0
```
```
# In any file in the /src
import Auth0 from 'react-native-auth0';
```
Draws the Following Error:
Failed to compile.

./node_modules/react-native-auth0/index.js
Module parse failed: Unexpected token (22:30)
You may need an appropriate loader to handle this file type.
|    */
|   constructor(options = {}) {
|     const { domain, clientId, ...extras } = options;
|     this.auth = new Auth({baseUrl: domain, clientId, ...extras});
|     this.webAuth = new WebAuth(this.auth);

**Comment by Widcket** [comment]
Hi @wutayng, I wasn't able to reproduce this issue. Here's what I did:
- `npm install -g create-react-native-web-app`
- `create-react-native-web-app ReactNativeWebTest`
- `cd ReactNativeWebTest `
- `yarn run web`
- `yarn add react-native-auth0`
- `import Auth0 from 'react-native-auth0';` in `App.js`.
- `yarn run web`

This produced the following output:
```
Compiled with warnings.

./src/App.js
  Line 12:  'Auth0' is defined but never used  no-unused-vars

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.
```

**Comment by wutayng** [comment]
Okay, I just reproduced your steps and got the same successful result. What I should have specified earlier is that the problem exists with the Example App linked on the CLI Github page (https://github.com/VISI-ONE/create-react-native-web-example)

Just now I cloned this example app, repeated your steps above, and got the original error. I have not been able to find any differences between this Example App and the original CLI App that should cause Auth0 not to compile.
